bundles/org.openhab.ui/web/src/components/widgets/system/oh-input.vue

```diff
@@ -2,7 +2,7 @@
   <f7-input v-if="!config.item || !config.sendButton" class="input-field" ref="input" v-bind="config" :value="(config.type.indexOf('date') === 0) ? valueForDatepicker : value" @input="$evt => updated($evt.target.value)" :change="updated" @calendar:change="updated" @texteditor:change="updated" @colorpicker:change="updated" />
   <f7-row no-gap v-else class="oh-input-with-send-button">
     <f7-input class="input-field col-90" ref="input" v-bind="config" :value="(config.type.indexOf('date') === 0) ? valueForDatepicker : value" @input="$evt => updated($evt.target.value)" :change="updated" @calendar:change="updated" @texteditor:change="updated" @colorpicker:change="updated" />
-    <f7-button class="send-button col-10" v-if="this.config.sendButton" @click="sendButtonClicked" v-bind="config.sendButtonConfig || { iconMaterial: 'done', iconColor: 'gray' }" />
+    <f7-button class="send-button col-10" v-if="this.config.sendButton" @click.stop="sendButtonClicked" v-bind="config.sendButtonConfig || { iconMaterial: 'done', iconColor: 'gray' }" />
```

Comment:
Could you elaborate why you added "stopPropagation" modifiers on all these widgets? (not that it doesn't make sense I just wanted to know why you made these changes, as it could have potential impacts unrelated to this PR)

Reply:
Without these modifiers, the user action performed on the control (toggle, button, stepper...) will at the same time toggle the accordion visibility. So they are definitely necessary, I hope I have not forgotten any.